Cargo.toml

```diff
@@ -50,6 +50,7 @@ petgraph = { version = "0.6.3", default-features = false }
 context-iterators = "0.2.0"
 serde_json = "1.0.97"
 delegate = "0.10.0"
+either = "1.9.0"
```

Comment:
There is an `Either` in `itertools` already, can you use that?

Reply:
Done, and re-exported it so we don't require `itertools` on the dependants.